### PR TITLE
Different types of software testing

### PR DESCRIPTION
**Pull request by andreagubellini**
Could not make it to test locally as hugo was giving me a few issues. Please check it by your side.

### CODE REVIEW

File: content/blog/eng/different-types-of-software-testing.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Unit are source-level tests which are executed on an individual portion of the application code: these tests are typically automated and run by software developers.
+Unit tests verify that the specific unit/function respects the intended design and works as expected.
+This type of tests is usually seen automated and executed in continuous integration servers.
+
+### Performance tests
+
+Performance tests verify if the application performance is compromised during high loads. This type of tests split in mupltiple sub-types:
+
+```
```
Comment (Jean85) [inline content/blog/eng/different-types-of-software-testing.md:76]: We should use the quoting format `> ` for this piece from Wiki

Comment (andreagubellini) [reply content/blog/eng/different-types-of-software-testing.md]: @Jean85 done

**Review by mar-gil — COMMENTED** [review]
First (and main) round of comments. It is very well written :)

**Comment by andreagubellini** [comment]
Hi guys,

After a few months of hold (sorry =P) I managed to add all the requested updates.

Please take a look and let me know!

**Pull request merged**